bigquery_etl/shredder/config.py

```diff
@@ -98,6 +99,11 @@ def fields(self) -> Tuple[str, ...]:
 FXA_SRC = DeleteSource(
     table="firefox_accounts_derived.fxa_delete_events_v1", field=USER_ID
 )
+REGRETS_SRC = DeleteSource(
+    table="regrets_reporter_stable.regrets_reporter_update_v1",
+    field="data_deletion_request.extension_installation_uuid",
+    conditions=("data_deletion_request IS NOT NULL",),
```

Comment:
Accidentally making the tuple into a string would be caught by the type annotations?

Reply:
it should, yeah.